doc/release-process.md

```diff
@@ -336,3 +336,23 @@ bitcoin.org (see below for bitcoin.org update instructions).
   - Optionally twitter, reddit /r/Bitcoin, ... but this will usually sort out itself
 
   - Celebrate
+
+### Additional information
+
+#### How to calculate `m_assumed_blockchain_size` and `m_assumed_chain_state_size`
+
+Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
+Note that all values values should be taken from a **fully synced** node and have an overhead of 5-10% added on top of its base value.
```

Comment:
Personally I really value values but I'm not valuing values so much that I think it should be repeated repeatedly.

Reply:
:rofl: thanks!